spaceship/lib/spaceship.rb

```diff
@@ -2,6 +2,7 @@
 require 'spaceship/base'
 require 'spaceship/client'
 require 'spaceship/launcher'
+require 'fastlane_core'
```

Comment:
There is no dependency from `spaceship` to `fastlane_core`, and I don't think it's a good idea to introduce one. The idea of `spaceship` is that people can build their own tools, that not necessarily use `fastlane`. Please remove the `require` statement and all references to `fastlane_core`